class Failure {
  final String message;
  // final StackTrace stackTrace;
  // final dynamic error;
  Failure([this.message = 'An exeption error occured']);
}
